packages/snaps-sdk/src/ui/components/button.ts

```diff
@@ -0,0 +1,67 @@
+import type { Infer } from 'superstruct';
+import { assign, literal, object, optional, string, union } from 'superstruct';
+
+import { enumValue } from '../../internals';
+import { createBuilder } from '../builder';
+import { LiteralStruct, NodeType } from '../nodes';
+
+export enum ButtonVariants {
+  Primary = 'primary',
+  Secondary = 'secondary',
+}
+
+export enum ButtonTypes {
```

Comment:
What is this supposed to do?

Reply:
This is to make a difference between a button that will submit a form and a button that should just emit an event to `onUserInput`.